config/locales/de.label.yml

```diff
@@ -111,3 +112,9 @@ de:
       :subtitle: Andalucia Ruby User Group
       :meta_desc: Andalucia Ruby User Group
       :custom_recurrence: letzter Dienstag des Monats um 19:30 Uhr
+    frankfurt:
+      :city: City
```

Comment:
These keys need to be updated I guess:
      name: Frankfurt.rb
      title: Frankfurt.rb
      subtitle: Ruby User Group Frankfurt
      meta_desc: Ruby / Rails User Group Frankfurt